mpdev/internal/apply/registry_test.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package apply
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestResolveFilePath(t *testing.T) {
+	r := newTestResource("testResource")
+
+	testCases := map[string]string{
+		// Absolute path
+		"/tmp/foo/2.txt": "/tmp/foo/2.txt",
+		// Relative path
+		"foo/2.txt": "dir/foo/2.txt",
```

Comment:
can you also add a test case leading `..` is used?

Reply:
Added